br/pkg/restore/snap_client/client.go

```diff
@@ -346,6 +363,8 @@ func (rc *SnapClient) InitCheckpoint(
 	snapshotCheckpointMetaManager checkpoint.SnapshotMetaManagerT,
 	config *pdutil.ClusterConfig,
 	logRestoredTS uint64,
+	hash []byte,
+	checkpointFirstRun bool,
```

Comment:
I refactor the code a bit in my last PR, now `!checkpointExists` is same as `checkpointFirstRun ` so we can remove `checkpointFirstRun`